_posts/2023-07-20-alert-anomaly-visual.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Visualizing alerts and anomalies using OpenSearch Dashboards visualization tools and applications"
+authors: 
+  - jdbright
+  - ashisagr
+  - ohltyler
+  - amgalitz
+  - hnyng
+  - vagimeli
+  - xeniatup
+date: 2023-07-20
+meta_keywords: alerting, anomaly detection, alert, anomaly, observabilty, dashboard, dashboards, visualizations, visualize
+meta_description: Integrating alerts and anomalies visualizations across OpenSearch Dashboards applications.
+twittercard:
+  description: Learn about integrating alerts and anomalies visualizations across OpenSearch Dashboards applications.
+category:
+  - community
+excerpt: 
+---
+
+# Introduction
+
+Prior to OpenSearch 2.9, users maintain state between the data they were viewing in the Discover and Dashboard interfaces and the data needed to create a new anomaly detector or alert monitor. Users provided feedback that creating new detectors or monitors was difficult because of context switching. With OpenSearch 2.9, users who use anomaly detection and alerting can create anomaly detectors and alerting monitors directly from their OpenSearch Dashboards [Vizlib](https://home.vizlib.com/) chart or line visualizations with select aggregations. After creating the new detector or monitor/alert, users can view anomalies or alerts overlaying the configured visualization. For users who have defined detectors or monitors, they can associate their existing detectors or monitors to the visualization, which helps track services that are not specific to the visualization, such as a dependent service.
+
+In this blog, you'll learn to create a new detector or monitor and add monitors and detectors to a visualization by way of the visualization, instead of through the Alerting or Anomaly Detection plugin pages.
+
+## Terms to know
+
+Here are terms you need to know:
+
+- _OpenSearch Dashboards visualizations_ are visualizations defined within a dashboard.
+- _Anomaly Detection plugin_ is the free OpenSearch plugin using the [Random Cut Forest algorithm](https://github.com/aws/random-cut-forest-by-aws) (that is, an unsupervised algorithm for detecting anomalous data points within a dataset) to detect anomalies in aggregated data.
+- _Alerting plugin_ is the free OpenSearch plugin used to monitor and trigger alerts on OpenSearch's health and machine-generated logs.
+
+## Getting started
+
+The alerting and anomaly detection visualizations feature is automatically enabled. You can turn it off by going to **OpenSearch Dashboards** > **Stack Management** > **Advanced Settings** and toggling off **Enable plugin augmentation**, which is in the **Visualization** pane. Before getting started with this feature, make sure you have:
```

Comment:
"The alerting and anomaly detection visualizations feature" is confusing, I think we can just say "The feature"